modules/Bio/EnsEMBL/Analysis/Hive/Config/TranscriptSelection.pm

```diff
@@ -0,0 +1,910 @@
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'layering_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_layer_' . $self->o('release_number'),
+      -host   => $self->o('layering_db_server'),
+      -port   => $self->o('layering_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'utr_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_utr_' . $self->o('release_number'),
+      -host   => $self->o('utr_db_server'),
+      -port   => $self->o('utr_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'genebuilder_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_gbuild_' . $self->o('release_number'),
+      -host   => $self->o('genebuilder_db_server'),
+      -port   => $self->o('genebuilder_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'pseudogene_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_pseudo_' . $self->o('release_number'),
+      -host   => $self->o('pseudogene_db_server'),
+      -port   => $self->o('pseudogene_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'ncrna_db' => {
+      -dbname => $self->o('ncrna_db_name'),
+      -host   => $self->o('ncrna_db_server'),
+      -port   => $self->o('ncrna_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'final_geneset_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_final_' . $self->o('release_number'),
+      -host   => $self->o('final_geneset_db_server'),
+      -port   => $self->o('final_geneset_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+  };
+}
+
+sub pipeline_create_commands {
+  my ($self) = @_;
+  return [
+    # inheriting database and hive tables' creation
+    @{ $self->SUPER::pipeline_create_commands },
+
+    'mkdir -p ' . $self->o('genome_dumps'),
+  ];
+}
+
+sub pipeline_wide_parameters {
+  my ($self) = @_;
+
+  return {
+    %{ $self->SUPER::pipeline_wide_parameters },
+    skip_projection     => $self->o('skip_projection'),
+    skip_rnaseq         => $self->o('skip_rnaseq'),
+    use_genome_flatfile => $self->o('use_genome_flatfile'),
+    genome_file         => $self->o('faidx_genome_file'),
+    }
+}
+
+## See diagram for pipeline structure
+sub pipeline_analyses {
+  my ($self) = @_;
+  my %commandline_params = (
+    'ncbi' => '-num_threads 3 -window_size 40',
+  );
```

Comment:
This idea of the hash is if someone want to use WU blast, they have the correct parameters. Could you please add the `'wu'` line from the original config?

Reply:
Done